fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -4,56 +4,52 @@ import com.gu.contentapi.client.model.Content
 import com.gu.facia.api.utils.{ResolvedMetaData, ItemKicker}
 import com.gu.facia.client.models.{SupportingItem, MetaDataCommonFields, Trail, TrailMetaData}
 
-case class ImageReplace(imageSrc: String, imageSrcWidth: String, imageSrcHeight: String)
-
-object ImageReplace {
-  def fromTrailMeta(trailMeta: MetaDataCommonFields): Option[ImageReplace] =
-    for {
-      imageReplace <- trailMeta.imageReplace.filter(identity)
-      imageSrc <- trailMeta.imageSrc
-      imageSrcWidth <- trailMeta.imageSrcWidth
-      imageSrcHeight <- trailMeta.imageSrcHeight
-    } yield ImageReplace(imageSrc, imageSrcWidth, imageSrcHeight)
-}
+case class FaciaImage(
+  imageType: ImageType,
+  imageSrc: String,
+  imageSrcWidth: Option[String],
+  imageSrcHeight: Option[String]
+)
+
+sealed trait ImageType
+case object Cutout extends ImageType { override def toString = "cutout" }
+case object Replace extends ImageType { override def toString = "replace" }
+case object Default extends ImageType { override def toString = "default" }
+
+object FaciaImage {
+
+  def getFaciaImage(maybeContent: Option[Content], trailMeta: MetaDataCommonFields): Option[FaciaImage] = {
```

Comment:
I already commented on this; pass in `ResolvedMetaData` instead of multiple resolutions of it. Don't know where my comment went and it still hasn't been done.

I will just do this today myself.